README.md

```diff
@@ -1,75 +1,28 @@
-# wait-for-it
+# wait-for-it.sh Github Action
 
-`wait-for-it.sh` is a pure bash script that will wait on the availability of a
-host and TCP port.  It is useful for synchronizing the spin-up of
-interdependent services, such as linked docker containers.  Since it is a pure
-bash script, it does not have any external dependencies.
-
-## Usage
+`wait-for-it.sh` is a pure bash script to wait on the availability of a host and TCP port.  
 
-```text
-wait-for-it.sh host:port [-s] [-t timeout] [-- command args]
```

Comment:
Why remove this? Adding a GitHub action is ok, but removing the current purpose of this script is against the role of this project.